### ISSUE TITLE
Database service unhealthy after first lando start

### ISSUE THREAD

**Issue by habhatti** [lando/lando#1719]
The database service starts only once on a new project and then database is unhealthy for all next starts in the laravel recipe. Issue persists even after destroying and starting.

Tell us about your setup
Windows 10 Pro (x64)
Lando v3.0.0-rc.18

docker version
Client: Docker Engine - Community
Version: 18.09.2
API version: 1.39
Go version: go1.10.8
Git commit: 6247962
Built: Sun Feb 10 04:12:31 2019
OS/Arch: windows/amd64
Experimental: false

Server: Docker Engine - Community
Engine:
Version: 18.09.2
API version: 1.39 (minimum version 1.12)
Go version: go1.10.6
Git commit: 6247962
Built: Sun Feb 10 04:13:06 2019
OS/Arch: linux/amd64
Experimental: false

Tell us about your .lando.yml
name: exampleproject
recipe: laravel
config:
webroot: app/public

Tell us about the command you were running
first run lando start - all is OK.
lando restart - see error below

Tell us about the error you got
info: Waiting until database service is ready...
warn: Service database is unhealthy
lando mysql
ERROR 2002 (HY000): Can't connect to local MySQL server through socket '/opt/bitnami/mysql/tmp/mysql.sock' (111)

**Comment by habhatti** [comment]
@pirog Hey there, if you need any assistance with narrowing this down, I'd be happy to help. This is a somewhat major issue in my case as it's a fresh install on Win.

**Comment by danepowell** [comment]
Probable dupe of #1718, I'm going to follow-up there but maintainers feel free to consolidate however you'd like.

**Comment by habhatti** [comment]
@danepowell Agreed, seems to be the same as #1718, only difference between that report is the platform. Will continue reporting on #1718.

**Comment by pirog** [comment]
agreed, closing in favor of #1718 and bumping that to the front of the backlog

**Comment by LPIX-11** [comment]
I think I found a solution for this
https://github.com/lando/lando/issues/1718#issuecomment-537113754
